File: C/Colleen.c
#include <stdio.h>
/*
comment1
*/
void func(void){int i = 1; i++; return;}
int main(void){
/*
comment2
*/
char *s="#include <stdio.h>%2$c%4$c%3$c%2$ccomment1%2$c%3$c%4$c%2$cvoid func(void){int i = 1; i++; return;}%2$cint main(void){%2$c%4$c%3$c%2$ccomment2%2$c%3$c%4$c%2$cchar *s=%5$c%1$s%5$c;printf(s,s,10,42,47,34);func();return(0);}%2$c";printf(s,s,10,42,47,34);func();return(0);}

File: PYTHON/Colleen.py
#comment1
def func():
  i = 1
def main():
  #comment2
  func()
  string = "#comment1{1}def func():{1}  i = 1{1}def main():{1}  #comment2{1}  func(){1}  string = {2}{0}{2}{1}  print(string.format(string, chr(10), chr(34))){1}main()"
  print(string.format(string, chr(10), chr(34)))
main()
